src/service/write-off-service.ts

```diff
@@ -0,0 +1,192 @@
+      updatedAt: writeOff.updatedAt.toISOString(),
+      to: parseUserToBaseResponse(writeOff.to, false),
+      amount: writeOff.amount.toObject(),
+    };
+  }
+
+  /**
+   * Parses a write-off object to a WriteOffResponse
+   * @param writeOff
+   */
+  public static asWriteOffResponse(writeOff: WriteOff): WriteOffResponse {
+    return {
+      ...this.asBaseWriteOffResponse(writeOff),
+      transfer: writeOff.transfer ? TransferService.asTransferResponse(writeOff.transfer) : undefined,
+    };
+  }
+
+  /**
+   * Returns all write-offs with options.
+   * @param filters - The filtering parameters.
+   * @param pagination - The pagination options.
+   * @returns {Array.<WriteOffResponse>} - all write-offs
+   */
+  public static async getWriteOffs(filters: WriteOffFilterParameters = {}, pagination: PaginationParameters = {}): Promise<PaginatedWriteOffResponse> {
+    const { take, skip } = pagination;
+
+    const options = this.getOptions(filters);
+    const [data, count] = await WriteOff.findAndCount({ ...options, take, skip });
+
+    const records = data.map((writeOff) => this.asWriteOffResponse(writeOff));
+
+    return {
+      _pagination: {
+        take, skip, count,
+      },
+      records,
+    };
+  }
+
+  // TODO: replace with ServerSettings
+  // @link https://github.com/GEWIS/sudosos-backend/issues/226
+  private static async getHighVATGroup(): Promise<VatGroup> {
+    const vatGroup = await VatGroup.findOne({ where: { percentage: 21, deleted: false, hidden: false } });
+    if (vatGroup) return vatGroup;
+    else throw new Error('High vat group not found');
+  }
+
+  /**
+   * Creates a write-off for the given user
+   * @param manager - The entity manager to use
+   * @param user - The user to create the write-off for
+   */
+  public static async createWriteOff(manager: EntityManager, user: User): Promise<WriteOffResponse> {
+    const balance = await BalanceService.getBalance(user.id);
+    if (balance.amount.amount > 0) {
+      throw new Error('User has balance, cannot create write off');
+    }
+
+    const amount = DineroTransformer.Instance.from(balance.amount.amount * -1);
+
+    const writeOff = Object.assign(new WriteOff(), {
+      to: user,
+      amount,
+    });
+
+    await manager.save(writeOff);
+    const transfer = await TransferService.createTransfer({
+      amount: {
+        amount: amount.getAmount(),
+        precision: amount.getPrecision(),
+        currency: amount.getCurrency(),
+      },
+      toId: user.id,
+      description: 'Write off',
+      fromId: null,
+    }, manager);
+
+    const highVatGroup = await WriteOffService.getHighVATGroup();
+    writeOff.transfer = transfer;
+    transfer.writeOff = writeOff;
+    transfer.vat = highVatGroup;
+
+    await manager.save(transfer);
+    await manager.save(writeOff);
+    return WriteOffService.asWriteOffResponse(writeOff);
+  }
+
+  public static async createAndCloseUser(user: User): Promise<WriteOffResponse> {
```

Comment:
Maybe rename to `createWriteOffAndCloseUser`, because now it can also mean that you create a user and immediately close it.